### PR TITLE
Add properties to type SubscriptionPricing

### PR DESCRIPTION
**Pull request by michaelk-redis**

### CODE REVIEW

File: types/responses/subscription.ts [hunk 1]
```diff
@@ -29,11 +29,20 @@ export type SubscriptionResponse = {
  * @param type The type of the subscription pricing
  * @param quantity The quantity of the subscription pricing
  * @param quantityMeasurement The quantity measurement of the subscription pricing
+ * @param typeDetails The details of the type measurement of the subscription pricing
+ * @param pricePerUnit The price per unit of the type measurement of the subscription pricing
+ * @param priceCurrency The currency of the price
+ * @param pricePeriod The time period of the price
+ *
```
Comment (danitseitlin) [inline types/responses/subscription.ts:36]: Please remove following blank line

File: types/task.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import { SubscriptionPricing } from "..";
```
Comment (danitseitlin) [inline types/task.ts:1]: Please import from the full path, rather than `".."`

**Review by danitseitlin — CHANGES_REQUESTED** [review]

File: types/responses/subscription.ts (hunk 1 above)
Comment (michaelk-redis) [reply types/responses/subscription.ts]: removed

File: types/task.ts (hunk 2 above)
Comment (michaelk-redis) [reply types/task.ts]: changed

**Pull request merged**